### ISSUE TITLE
[Question] How to shift+click in GUI_NORMAL?

### ISSUE THREAD

**Issue by gilbh** [pit-ray/win-vind#69]
In GUI_NORMAL, which mouse over a link, I tried `shift+o` for shift click (opening a link in a new browser page), but I don't think it worked because the link opened in the same window.

Is there a way to do this?

Thanks!

**Comment by pit-ray** [comment]
There is currently no such function. However, from the Issue you raised, I consider that the ability to pass any key is necessary.

**Comment by gilbh** [comment]
Yes,  I agree that that would be very useful. I using win-vind a lot and am loving it! Keep up the excellent work.
